app/src/main/res/values-night/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="color_highlight">#4d2c91</color>
     <color name="color_mention">#6e3737</color>
     <color name="color_reward">#00766c</color>
+    <color name="ripple_color">#cca0a0a0</color>
```

Comment:
You don't need to add the color here if it's the same value for dark & light mode